src/searchForm/story.tsx

```diff
@@ -115,3 +117,9 @@ stories.add('StepperSection', () => (
 ))
 stories.add('AutoCompleteOverlay', () => <AutoCompleteExample component={AutoCompleteOverlay} />)
 stories.add('AutoCompleteSection', () => <AutoCompleteExample component={AutoCompleteSection} />)
+
+stories.add('SearchForm', () => (
+  <MediaSizeProvider>
```

Comment:
Or are you gonna use the searchForm ? in a new custom section ?

Reply:
The goal is to use it in a hero section